junit/src/main/java/io/cucumber/junit/JUnitCucumberOptionsProvider.java

```diff
@@ -67,14 +68,12 @@ public boolean monochrome() {
 
         @Override
         public SnippetType snippets() {
-            switch (annotation.snippets()) {
-                case UNDERSCORE:
-                    return SnippetType.UNDERSCORE;
-                case CAMELCASE:
-                    return SnippetType.CAMELCASE;
-                default:
-                    throw new IllegalArgumentException("" + annotation.snippets());
-            }
+            return annotation.snippets();
```

Comment:
Why did you also change this?

Reply:
Ah. I see. `io.cucumber.core.snippets.SnippetType` is still part of the public API. It shouldn't be. The local duplicates keep these sort of implementation details away from the user.

Reply:
Can you instead revert this change and remove the `@API` annotation from `io.cucumber.core.snippets.SnippetType`?

Reply:
Not sure why this is better. Anyway, I reverted the changes for the JUnit and TestNG CucumberOptions annotations (the part about snippets) and removed the `@API` annotation from `io.cucumber.core.snippets.SnippetType`.

Reply:
Cheers.

Currently a user needs to declare two dependencies to user cucumber. For example `cucumber-java` and `cucumber-junit`. The transitive dependencies are automatically included by dependency management tools. 

When designing a library it is a good practice not to expose classes from transitive dependencies to the consumer.  This makes versioning easier. `SnippetType` is defined in `core`. By having an alias in `cucumber-junit` we avoid exposing core.